runtime/src/iree/vm/shims.h

```diff
@@ -259,6 +268,24 @@ IREE_VM_ABI_FIXED_STRUCT(rII, {
   int64_t i2;
 });
 
+IREE_VM_ABI_FIXED_STRUCT(rIIrIIrIIIIIffi, {
```

Comment:
for some of the more esoteric ones we could define them inline in the module.c - anywhere at the top should be fine - here is mostly useful for reusable things but when this bespoke (unlikely to ever be used) it seems nicer to keep them local (one day I hope to have these auto generated from the vm.imports ops, anyway)